docs/source/overview.rst

```diff
@@ -0,0 +1,94 @@
+
+It's easiest to describe The MozDef Event Framework as a set of micro-services you can use to integrate event sources with the `Mozilla Enterprise Defense platform <https://mozdef.readthedocs.io/en/latest/>`_ (MozDef).
+
+Provides
+--------
+
+Many resources are not wholly contained within the datacenter, there are SAAS and IAAS providers which may provide event logging that is external to the network where your SIEM resides. 
+Exposing your security infrastructure doesn't ensure a secure pipeline. In order to solve for this we decided on pulling that information as our method of choice, as opposed to pushing.
+Utilizing cloud based microservices can reduce exposure, ensure efficiency and scalability, and leads to less maintenance.
+
+We created a framework using AWS Cloudformation and the Serverless framework to build a pipeline that will allow placement of lambda scripts specific for an event source and provide continuous deployment and integration.
+This takes the guesswork out of having to write serverless and cloudformation code every time  you want to deploy a pipeline for a new event source and creates a standard method to be used to obtain those events. 
+
+Webhook sources utilize an AWS API gateway that they can post events to. As we scale this out the capabilities will increase.
+The hope is to have a scalable framework that can be deployed for both rest and webhook based sources.
+
+Flow
+----
+
+.. image:: images/MozDef-Event-Framework_process_flow.png
+   :width: 600
+   :alt: High Level Overview of the Work/Process Flow Steps From Start to Finish
+
+Components
+------------
+The following components make up this framework:
+
+
+   * AWS API Gateway
+   * AWS Cloudformation
+   * AWS CloudWatch
+   * AWS CodePipeline
+   * AWS CodeCommit
+   * AWS CodeBuild
+   * AWS Lambda
+   * AWS S3
+   * AWS SQS
+   * AWS XRay
+   * GitHub
+   * Serverless framework
+
+Status
+------
+
+The MozDef event framework is under development at this time.
+
+Goals
+-----
+
+High level
+**********
+
+* Provide a platform for use by security infrastructure engineers to rapidly deploy a pipeline to enable ingestion of events into MozDef.
+* Facilitate continuous integration and development.
+* Facilitate repeatable, predictable processes for adding new event sources.
+* Provide a means with which to reprocess any events that do not meet the requirements you set.
+
+Technical
+*********
+
+* Offer micro services that enable rapid consumption of various event sources as needed.
+* Scalable, should be able to handle thousands of events per second, provide validation, and a means to reprocess events that fail validation by utilizing the CI/CD pipeline this framework builds.
+
+Roadmap
+-------
+
+Done
+****
+
+   * Allows the use of Webhook API connectivity
+   * Can pull configuration from one or more sources (Github/CodeCommit) during build by selecting the appropriate CF template
+   * Utilizes ssm/keystore parameters to prevent exposure of secrets
```

Comment:
Sorry to nitpick, but I think the correct term is SSM Parameter Store.

Reply:
the / represents or, meaning either can be used here.

Reply:
Per our conversation I'll fix.

Reply:
resolved by changing to SSM and Secrets manager parameters.